src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  padding-inline: 16px;
```

Comment:
Consider using relative units (like percentages or ems) for padding to improve responsiveness.